api/blueprints/routes.py

```diff
@@ -26,25 +26,25 @@ def object_as_dict(obj):
 @bp.route(f'{API_BASE_PATH}/characters/<string:character_id>', methods=['GET'])
 def get_character(character_id):
     character = Character.query.get_or_404(character_id)
-    return jsonify({'character': object_as_dict(character)})
+    return jsonify({'results': object_as_dict(character)})
 
 
 @bp.route(f'{API_BASE_PATH}/conversations/<int:conversation_id>', methods=['GET'])
 def get_conversation(conversation_id):
     conversation = Conversation.query.get_or_404(conversation_id)
-    return jsonify({'conversation': object_as_dict(conversation)})
+    return jsonify({'results': object_as_dict(conversation)})
 
 
 @bp.route(f'{API_BASE_PATH}/genres/<int:genre_id>', methods=['GET'])
 def get_genre(genre_id):
     genre = Genre.query.get_or_404(genre_id)
-    return jsonify({'genre': object_as_dict(genre)})
+    return jsonify({'results': object_as_dict(genre)})
```

Comment:
You can return `object_as_dict(genre)` as JSON directly in the case of a single resource, it's a common practice